docs/spec.md

```diff
@@ -5077,21 +5076,41 @@ dimension `d` in `operand`.
 
 #### Semantics
 
-Sorts a variadic number of tensors in `inputs` together, according to a custom
-`comparator`, along the given `dimension` and produces a variadic number of
-tensors as `results`. If `is_stable` is true, then the sorting is stable, that
-is, relative order of elements considered to be equal by the comparator is
-preserved. Two elements `e1` and `e2` are considered to be equal by the
-comparator if and only if `comparator(e1, e2) = comparator(e2, e1) = false`.
-
-More formally, for all `0 <= id < jd < dim(inputs[0], d)`, either
-`compare_i_j = compare_j_i = false` or `compare_i_j = true`, where:
-
-1. `compare_i_j` $=$
-   `comparator(inputs[0][i], inputs[0][j], inputs[1][i], inputs[1][j], ...)`.
-1. For all indices `i = [i0, ..., iR-1]` and `j = [j0, ..., jR-1]`.
-1. Where `i` $=$ `j` everywhere except for the `d`th dimension.
-1. Where `d` $=$ `dimension >= 0 ? dimension : rank(inputs[0]) + dimension`.
+Sorts 1-dimensional slices of `inputs` along the dimension `dimension` together,
```

Comment:
I think that it's good to be explicit about what the op does - "sorts 1-dimensional slices" - similarly to e.g. ConvolutionOp, because I forgot this, and it took me a while to recall.